python/plugins/processing/algs/r/RUtils.py

```diff
@@ -45,6 +45,8 @@ class RUtils(object):
     R_USE64 = 'R_USE64'
     R_LIBS_USER = 'R_LIBS_USER'
 
+    rscriptfilename = userFolder() + os.sep + 'processing_script.r'
```

Comment:
I know it, it's just a copy-paste and I was a bit lazy (don't want to verify all the R files)